how-to/setup_dashboard.rst

```diff
@@ -0,0 +1,50 @@
+JAAS: Setup Juju dashboard
+==========================
+
+Juju dashboard is a web UI that is intended to supplement the CLI experience with aggregate views and at a glance health checks.
+
+This how-to provides you with instructions on how to setup Juju Dashboard for your JAAS deployment.
+
+.. hint::
+    To explore Juju Dashboard features you can go `here <https://juju.is/docs/juju/the-juju-dashboard>`_.
+
+
+Prerequisites
+-------------
+
+For this how-to you will need the following:
+
+- A running JAAS environment, see :doc:`our tutorial <../tutorial/deploy_jaas_microk8s>`.
+
+Deploy Juju Dashboard
+---------------------
+First deploy the Juju Dashboard charm.
+
+.. code:: bash
+
+    juju switch <model_where_jimm_is>
+    juju deploy juju-dashboard-k8s dashboard
+    juju integrate dashboard jimm-app
+
+Then you need to expose your dashboard through an ingress.
+
+.. hint::
+    You can follow :doc:`this guide <./setup_ingress_with_tls>`.
```

Comment:
Should we mention "for setting up ingress with TLS" (explaining why you might want to go to the link even though we just explained how to setup ingress below.
 ```suggestion
    You can follow :doc:`this guide <./setup_ingress_with_tls>` to add TLS to your ingress.
```